src/web/client/common/localStore.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+'use strict';
+import * as vscode from "vscode";
+import { getHeader } from "./authenticationProvider";
+import { PORTAL_LANGUAGES, PORTAL_LANGUAGES_URL_KEY, PORTAL_LANGUAGE_DEFAULT, WEBPAGEID_URL_KEY, WEBPAGES, WEBSITE_LANGUAGES, WEBSITE_LANGUAGES_URL_KEY } from "./constants";
+import { readSchema } from "./portalSchemaReader";
+
+let orgMap = new Map()
+let webpagestowebpagesId = new Map();
+let languageIdCodeMap = new Map();
+let websiteIdtoLanguage = new Map();
+
+
+export async function languageIdtoCodeMap(accessToken: string, dataverseOrg: any, entity: string) {
+
+    try {
+        const requestUrl = getCustomRequestURL(dataverseOrg, PORTAL_LANGUAGES, PORTAL_LANGUAGES_URL_KEY);
+        const req = await fetch(requestUrl, {
```

Comment:
I guess this is response, please rename.

Reply:
done